config/i18n.json

```
@@ -583,6 +583,25 @@
     "MissingClassDescription": "The build you're trying to view is for a character class you don't have.",
     "NewEmptyLoadout": "Create Empty Loadout",
     "NoBuildsFound": "No builds found. Try expanding your filtering criteria.",
+    "NoBuildsFoundExplainer": {
+      "Header": "No builds found. Here are possible reasons DIM couldn't find any sets:",
+      "AlwaysInvalidMods": "These mods don't fit into any of your items:",
+      "RemoveMods": "Remove these mods",
+      "MaybeRemoveMods": "Consider removing mods",
+      "AssumptionsRestricted": "DIM is restricted in its allowed assumptions about armor energy",
+      "AssumeElementChange": "Assume armor will change energy type",
+      "AssumeMasterworked": "Assume armor is masterworked",
+      "ActiveSearchQuery": "An active search query is restricting the items DIM is considering for builds",
+      "MaybeRemoveSearchQuery": "Consider clearing your search query",
+      "MaybeAllowMoreItems": "Consider allowing more items",
+      "BadSlot": "In the {{bucketName}} slot, none of the considered items could accommodate these mods:",
+      "LowerBoundsFailed": "Many sets did not meet minimum stat tier requirements",
+      "UpperBoundsFailed": "Many sets did not meet maximum stat tier requirements",
+      "ModAssignmentFailed": "Many sets could not accommodate all requested mods",
+      "MaybeDecreaseLowerBounds": "Consider reducing minimum stat tier requirements",
+      "MaybeIncreaseUpperBounds": "Consider increasing maximum stat tier requirements",
+      "AllowAutoStatMods": "Allow DIM to automatically pick stat mods"
```

```suggestion
      "AllowAutoStatMods": "Allow DIM to automatically include stat mods"
```
speaking of previous critiques of the word "pick", "pick" has a strong connotation of choosing *between* available things, instead of choosing to do them at all.